### PR TITLE
Header section

### PR DESCRIPTION
**Pull request by Tomikander**
I can’t run the code in the IDE, and the pictures don’t appear there like in the video, that is, what I’m writing in the code is not there, that is, there should be a background there and it’s not there... Look at the code, maybe I did something wrong in the video?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
-    <head></head>
-    <body>
-        Hello, World
-    </body>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Simple Landing Page</title>
+    <meta name="viewport" content="width=device-width, initian-scale=1">
+    <link rel="stylesheet" hret="style.css">
```
Comment (RomanKontsevoi) [inline index.html:8]: You have a typo here @Tomikander

**Review by RomanKontsevoi — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+           <nav>
+            <img src="img/logo.png"   class="logo">
+                <ul>
+                    <li><a href="#home">Home</a></li>
+                    <li><a href="#how">How it work</a></li>
+                    <li><a href="#organic food">organic food</a></li>
+                    <li><a href="#about">About farm</a></li>
+                    <li><a href="#animals">Animals</a></li>
+                    <li><a href="@products">Products</a></li>
+                    <li><a href="contacts">ontacts</a></li>
```
Comment (RomanKontsevoi) [inline index.html:25]: Pay attention to the last two rows, also typos

File: index.html (hunk 1 above)
Comment (Tomikander) [reply index.html:8]: What's going on here? Am I a little confused?

File: index.html (hunk 2 above)
Comment (Tomikander) [reply index.html:25]: I found it, everything worked out:)

**Pull request merged**